src/main/resources/application.yml

```diff
@@ -70,8 +78,8 @@ initializr:
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
-            version: 2.0.0.RELEASE
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
```

Comment:
We dropped aggregate-counter in the 2.1.0.RELEASE.

Reply:
`- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`